python/tvm/relay/frontend/tflite.py

```diff
@@ -1318,7 +1317,9 @@ def convert_conv(self, op, conv_type):
         if is_depthwise_conv:
```

Comment:
Can we move the condition of input channels == 1 to where we compute `is_depthwise_conv`